indico/modules/events/editing/client/js/actions.js

```diff
@@ -5,41 +5,29 @@
 // modify it under the terms of the MIT License; see the
 // LICENSE file for more details.
 
-import editableDetailsURL from 'indico-url:event_editing.api_editable';
-import getReviewURL from 'indico-url:event_editing.api_review_editable';
-
 import {indicoAxios} from 'indico/utils/axios';
 import {ajaxAction, submitFormAction} from 'indico/utils/redux';
 
 export const SET_LOADING = 'SET_LOADING';
 export const SET_DETAILS = 'SET_DETAILS';
 
-export function loadTimeline(eventId, contributionId, type) {
-  const url = editableDetailsURL({confId: eventId, contrib_id: contributionId, type});
-  return ajaxAction(() => indicoAxios.get(url), SET_LOADING, SET_DETAILS);
+export function loadTimeline() {
+  return async (dispatch, getStore) => {
+    const url = getStore().staticData.editableDetailsURL;
```

Comment:
You can still use the selector here: `getStaticDate(getStore())`

Reply:
Ahh, I forgot to change it after I tested it (this was meant to make the whole thing testable)